### PR TITLE
Bugfix/atr 789 dev change acumatica logo

### PR DESCRIPTION
**Pull request by SENya1990**
**Merge after https://github.com/Acumatica/Acuminator/pull/516**

Overview:
- Updated Acumatica logo in all places.

### CODE REVIEW

**Comment by SENya1990** [comment]
@dropsonic maybe we should not update the logo for icons in VS menu? IMHO it looks a bit better with the old logo because it is simpler and the icon is small. Compare this:

_New logo:_
image

_Old logo:_
image

File: src/Acuminator/Acuminator.Analyzers/packageIcon.png
Comment (dropsonic) [inline src/Acuminator/Acuminator.Analyzers/packageIcon.png:1]: The original one was `256x256`, the new one is not even rectangular, it is `2000x1973`. Is it OK with NuGet? I suppose we should follow its recommendations.

Comment (SENya1990) [reply src/Acuminator/Acuminator.Analyzers/packageIcon.png:1]: Yes, we probably should do it. Do we need to ask the creator of the icon for the 256x256 resolution, or some online resizer will be ok?

Comment (dropsonic) [reply src/Acuminator/Acuminator.Analyzers/packageIcon.png:1]: Yes, please write to Drew Donaghy and request an icon.

Comment (dropsonic) [reply src/Acuminator/Acuminator.Analyzers/packageIcon.png:1]: Yes, please write to Drew Donaghy and request an icon.

Comment (SENya1990) [reply src/Acuminator/Acuminator.Analyzers/packageIcon.png:1]: Received an icon from Drew Donaghy and replaced this icon with it.

**Comment by dropsonic** [comment]
> @dropsonic maybe we should not update the logo for icons in VS menu? IMHO it looks a bit better with the old logo because it is simpler and the icon is small. Compare this:
> 
> _New logo:_ image
> 
> _Old logo:_ image

Please consult with Drew on that.

**Comment by SENya1990** [comment]
@dropsonic got a reply that we should use the new logo everywhere. So, we can't keep the old logo.

**Review by dropsonic — APPROVED** [review]

**Pull request merged**